src/components/ThemeModeSwitcher/index.jsx

```diff
@@ -0,0 +1,25 @@
+/* Temporary component for testing theme implementation and dark mode.  Used in the right side
+nav menu currently.
+
+@TODO convert to some final implementation
+*/
+import { useEffect, useState } from 'react';
```

Comment:
This looks like an unused import. Was it added purposefully?

Reply:
Nope, got automatically added when we were experimenting, I'll get rid of it :+1: